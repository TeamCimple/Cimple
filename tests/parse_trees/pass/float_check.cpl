int a = 345.89;

int test()
{
	return 1;
}

int main(int argc)
{
	return 0;
}
