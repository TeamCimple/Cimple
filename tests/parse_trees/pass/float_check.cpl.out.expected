FuncDecl(Name: Var(main) ReturnType: int Parameters: PARAM(int Var(argc))  Body: CompoundStatement(Declarations: (float EQUALS(Var(pi) Float(3.14),), (float EQUALS(Var(a) PLUS(Identifier(pi), Float(2.)),),  StatementList: RETURN(Int(0)))) 
